spark-connector-examples/src/main/scala/io/pravega/example/spark/StreamPravegaToConsole.scala

```diff
@@ -20,11 +20,13 @@ object StreamPravegaToConsole {
       .getOrCreate()
 
     val scope = sys.env.getOrElse("PRAVEGA_SCOPE", "examples")
-    val controller = sys.env.getOrElse("PRAVEGA_CONTROLLER", "tcp://127.0.0.1:9090")
+    val controller = sys.env.getOrElse("PRAVEGA_CONTROLLER_URI", "tcp://127.0.0.1:9090")
+    val checkpointLocation = sys.env.getOrElse("CHECKPOINT_DIR", "/tmp/spark-checkpoints-StreamPravegaToConsole")
 
     spark
       .readStream
       .format("pravega")
+      .option("allow_create_scope", "false")
```

Comment:
As we disable this option here, it will need some extra effort for users to create the scope manually. Should we address this in the doc?

Reply:
This is a good point. I will change this to look at the existence of the PROJECT_NAME environment variable. If it is exists, we can assume SDP.

Reply:
This has been fixed.